candidate_models/model_commitments/__init__.py

```diff
@@ -1,10 +1,13 @@
+from candidate_models.base_models import base_model_pool
+
 from candidate_models.model_commitments.cornets import cornet_brain_pool
-from candidate_models.model_commitments.ml_pool import ml_brain_pool
-from candidate_models.utils import UniqueKeyDict
+from candidate_models.model_commitments.model_layer_def import model_layers
+from submission.ml_pool import MLBrainPool
+from submission.utils import UniqueKeyDict
```

Comment:
Everytime I see a `submission` import, I confuse it with a local import from this repository. I'm really thinking more and more that it should be a package of `brainscore`, i.e. `brainscore.submission`.